### ISSUE TITLE
How to use

### ISSUE THREAD

**Issue by neubreed** [angrychimp/php-dkim#6]
Are you able to add usage docs please?

Thanks a lot

**Comment by angrychimp** [comment]
Certainly. I'll see if I can put something together in the next few days.

**Comment by vivex** [comment]
@angrychimp  Few months gone :smile_cat:  , please put it in wiki or in README

**Comment by turboflash** [comment]
To anyone who may find it useful, you can just change the following lines in DKIM_Verify::validate()
`        $signatures = $this->_getHeaderFromRaw('DKIM-Signature');
        $signatures = $signatures['DKIM-Signature'];`
to
`        $signatures = $this->_getHeaderFromRaw('X-Google-DKIM-Signature');
        $signatures = $signatures['X-Google-DKIM-Signature'];`
so that you can do validation on original message with X-Google-DKIM-Signature.

**Comment by turboflash** [comment]
One more thing, for anyone who wants to do a quick verification on the email message authenticity, here is the code.
`require_once "DKIM/Verify.php";

$email = file_get_contents($argv[1]);
$dkim_verify = new DKIM_Verify($email);
$r = $dkim_verify->validate();
print_r($r);
`

When running the script, just supply the filename as the argument. :wink:
